default.json

```diff
@@ -1,22 +1,60 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:base",
+    "group:typescript-eslintMonorepo",
+    "group:commitlintMonorepo"
   ],
+  "timezone": "America/Los_Angeles",
   "prHourlyLimit": 0,
-  "masterIssue": true,
   "lockFileMaintenance": {
-    "schedule": [
-      "before 9am on monday"
+    "enabled": true,
+    "extends": [
+      "schedule:weekly"
     ]
   },
   "packageRules": [
     {
-      "description": "Automerge non-major updates",
-      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
-      "automerge": true,
-      "automergeType": "pr",
-      "automergeStrategy": "squash"
+      "description": "Label NPM dev dependencies",
+      "matchDatasources": ["npm"],
+      "matchDepTypes": ["devDependencies"],
+      "labels": ["devDependencies"]
+    },
+    {
+      "description": "Label NPM dependencies",
+      "matchDatasources": ["npm"],
+      "matchDepTypes": ["dependencies"],
+      "labels": ["dependencies"]
+    },
+    {
+      "description": "Configure Github Actions dependencies (labels, commit format)",
+      "matchDepTypes": ["action"],
+      "labels": ["actions"],
+      "semanticCommitType": "chore",
+      "semanticCommitScope": "ci-deps"
+    },
+    {
+      "description": "Require stability of NPM dependencies which are not managed by Side Inc. (since they can be unpublished)",
+      "matchDatasources": ["npm"],
+      "matchDepTypes": ["dependencies"],
+      "excludePackagePatterns": ["^@side"],
+      "stabilityDays": 3
+    },
+    {
+      "description": "Skip faker updates (since it is no longer supported)",
+      "matchPackageNames": ["faker", "@types/faker"],
+      "enabled": false
+    },
+    {
+      "matchPackageNames": ["config", "@types/config"],
+      "groupName": "Node config and types",
+      "groupSlug": "node-config"
+    },
+    {
+      "description": "Test Library Monorepo group",
```

Comment:
Maybe also include lint-config libs as a group?

Reply:
Oh good call - I also just noticed a number of the monorepo groups I included are included by default from config:base which pulls [`group:monorepos`](https://docs.renovatebot.com/presets-group/#groupmonorepos) so I'll remove those